### PR TITLE
Fix cut acceptance plotting

### PR DESCRIPTION
**Pull request by ghusheng**
Fix the `n_minus_1 plotting` option in the `get_cut_eff`

### CODE REVIEW

**Review by FaroutYLq — COMMENTED** [review]
Thanks @ghusheng for catching this. but I feel like we can just keep using `cut_func` as before to make the codes concise. I made a commit. Can you please check if it makes sense to you?

File: saltax/match/utils.py [hunk 1]
```diff
@@ -698,8 +698,9 @@ def get_cut_eff(
         result_dict["all_cuts"][i] = len(selected_events_all_cut) / len(selected_events)
         result_dict["all_cuts_upper"][i] = interval.high
         result_dict["all_cuts_lower"][i] = interval.low
+
         for cut_oi in all_cut_list:
-            selected_events_cut_oi = selected_events[apply_single_cut(selected_events, cut_oi)]
+            selected_events_cut_oi = selected_events[cut_func(selected_events, cut_oi)]
             # Efficiency curves with Clopper-Pearson uncertainty estimation
             interval = binomtest(len(selected_events_cut_oi), len(selected_events)).proportion_ci()
             result_dict[cut_oi][i] = len(selected_events_cut_oi) / len(selected_events)
```
Comment (ghusheng) [inline saltax/match/utils.py:706]: Hi Lanqing! If we still keep the code like this, I am not sure the N-1 cut acceptance is correct.

Comment (FaroutYLq) [reply saltax/match/utils.py]: Great thanks for the sanity check now I wake up. Is it better now?

Comment (ghusheng) [reply saltax/match/utils.py]: Haha, the thing is that if you change in this way then it won't work for the single cut acceptance. These two need to be treated differently :)

Comment (FaroutYLq) [reply saltax/match/utils.py]: You are right. I buy it. Let me revert.

**Review by FaroutYLq — APPROVED** [review]
Thanks again @ghusheng for capturing this important bug. Now it should be fixed.

**Comment by ghusheng** [comment]
> Thanks again @ghusheng for capturing this important bug. Now it should be fixed.

np

**Pull request merged**